web/Views/Landlord/Profile.cshtml

```diff
@@ -63,6 +63,13 @@
     </div>
     
     <h4>ChangeAhead Charter</h4>
-    @*TODO: Database does not include this in this branch yet.*@
+    @if (Model.LandlordProvidedCharterStatus)

```

Comment:
I think this should actually use `LandlordStatus` mainly. Worth chatting to Tom + Blanca who added the two columns for what their intention was, but I think it's:

* LandlordStatus - whether the landlord has been approved by an admin
    * Which is the main thing we care about
* LandlordProvidedCharterStatus - whether the landlord said they have already signed the charter
    * Which I think is just intended to be useful for admins to see so they know whether to check their records, or start the approval process

Reply:
I've just spoken to them:
- 'LandlordStatus' is a separate thing, referring to the type of landlord (they got it from the existing Landlord Registration Form [Form for Landlords to Complete: (Charities/RSL/PSL/LA/Agencies)](https://docs.google.com/forms/d/e/1FAIpQLSesM0lm2YyY95B8F_XtfakZciti6jDnDhlGAgKIWPqAUZfDkw/viewform?vc=0&c=0&w=1&flr=0))
- 'LandlordProvidedCharterStatus' is indeed whether the landlord said they have signed the charter
- a relevant 'CharterStatus' field (contrary to what was suggested on the ticket) doesn't actually yet exist, so I'll go create that now

Reply:
That makes sense, thanks. Would you mind adding comments to those fields in the DB model to explain what they are? (can be separate PR if you want to get this merged)

Reply:
Comments added (I didn't think merging this was urgent, so I did it on this pull request)